packages/client/lib/commands/HEXPIRE.ts

```diff
@@ -0,0 +1,36 @@
+import { Command, NumberReply, ArrayReply, RedisArgument } from "../RESP/types";
+import { RedisVariadicArgument, pushVariadicArgument } from "./generic-transformers";
+
+export const HASH_EXPIRATION = {
+  /** The field does not exist */
+  FIELD_NOT_EXISTS: -2,
+  /** Specified NX | XX | GT | LT condition not met */
+  CONDITION_NOT_MET: 0,
+  /** Expiration time was set or updated */
+  UPDATED: 1,
+  /** Field deleted because the specified expiration time is in the past */
+  DELETED: 2
+} as const;
+  
+export type HashExpiration = typeof HASH_EXPIRATION[keyof typeof HASH_EXPIRATION];
+
+export type HashExpirationReply = NumberReply<HashExpiration>;
+
+export default {
+  FIRST_KEY_INDEX: 1,
+  transformArguments(
+    key: RedisArgument, 
+    fields: RedisVariadicArgument,
+    seconds: number,
+    mode?: 'NX' | 'XX' | 'GT' | 'LT',
```

Comment:
Should we do
```typescript
interface HExpireOptions {
  mode?: 'NX' | 'XX' | 'GT' | 'LT';
}
```
for "future proofing" (in case there will be more optional arguments)

Reply:
it doesn't really future proof, it just makes it easier if these options are shared between multiple commands.  Unsure it adds that much.